core/runtime/runtime_api/impl/parachain_host.cpp

```diff
@@ -291,4 +292,11 @@ namespace kagome::runtime {
                                      "ParachainHost_minimum_backing_votes");
   }
 
+  outcome::result<std::vector<ValidatorIndex>>
+  ParachainHostImpl::disabled_validators(const primitives::BlockHash &block) {
+    OUTCOME_TRY(ctx, executor_->ctx().ephemeralAt(block));
+    return executor_->call<std::vector<ValidatorIndex>>(
```

Comment:
[Should missing method cause error?](https://github.com/qdrvm/kagome/blob/ab4c7c42adce9d4a499153e5d3c7a0aea4bd07f6/core/runtime/runtime_api/impl/beefy.cpp#L26)

Reply:
Should not. I'll rewrite it to make particular.